src/compose/volume.ts

```diff
@@ -54,6 +48,7 @@ export class Volume {
 		// Convert the docker inspect to the config
 		const config: VolumeConfig = {
 			labels: inspect.Labels || {},
+			driver: inspect.Driver,
```

Comment:
I can't fully remember, but `local` is *always* returned by the daemon when nothing is passed, right?

Reply:
every volume has a driver; local is just the default one, so I would expect Dockerode to return local here. It's not an optional value in the types...

Reply:
Yeah I figured, just wanted to make sure. The tests will catch this anyway.